jbcli/jbcli/tests/test_cli.py

```diff
@@ -12,7 +12,7 @@
 from ..cli.jb import DEVLANDIA_DIR, cli
 
 
-CORE_DIR = os.path.join(DEVLANDIA_DIR, 'environments/core')
+CORE_DIR = DEVLANDIA_DIR
```

Comment:
Let's change all the references to CORE_DIR to reference DEVLANDIA_DIR and get rid of CORE_DIR